Palindrome.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Pallindrome {
+
+    public static void main(String[] args) {
+        int n=1231;
```

Comment:
Can you use scanner to take the user input, If not its all good. this still works.

Reply:
Yes definitely. I will add that.